<launch>
  <node name="flashgo_node"  pkg="flashgo"  type="flashgo_node" output="screen">
    <param name="port"         type="string" value="/dev/flashlidar"/>  
    <param name="baudrate"     type="int"    value="115200"/>
    <param name="frame_id"     type="string" value="laser_frame"/>
    <param name="angle_fixed"  type="bool"   value="true"/>
    <param name="angle_min"    type="double" value="-180" />
    <param name="angle_max"    type="double" value="180" />
    <param name="ignore_array" type="string" value="" />
  </node>
  <node pkg="tf" type="static_transform_publisher" name="base_link_to_laser4"
    args="0.2245 0.0 0.2 0.08 0.0  0.0 /base_footprint /laser_frame 40" />
</launch>
